docs/release-notes.txt

```diff
@@ -4,6 +4,115 @@
  Release Notes
 ###############
 
+**************
+ Version 0.17
+**************
+
+Version 0.17.0
+==============
+
+**Release Date:** September 28, 2021
+
+**New Features**
+
+-  Support the `MMDetection <https://github.com/open-mmlab/mmdetection>`__ library in Model Hub to
+   easily train object detection models. MMDetection provides efficient implementations of popular
+   object detection methods like Mask R-CNN, Faster R-CNN, and DETR on par with Detectron2. In
+   addition, cutting-edge approaches from academia are regularly added to the library.
+
+-  Add a new ``--auto-work-dir`` feature to ``det deploy local``. Setting ``--auto-work-dir
+   /some/path`` will have two effects: first, ``/some/path`` will be bind-mounted into the container
+   (still as ``/some/path``); second, interactive jobs (notebooks, shells, and commands) will run in
+   the provided working directory by default. Note that containers run as the root user by default,
+   so you may want to configure your user with ``det user`` such that interactive jobs run as your
+   regular user.
+
+-  Add the ability to use customizable master configuration templates in ``det deploy aws|gcp``.
+
+-  WebUI: Allow experiment owners to delete their own experiments.
+
+   .. image:: https://user-images.githubusercontent.com/220971/134048799-cd663a75-cb24-4f44-9a8a-c2ff23222cef.png
+
+-  Support configuring an idle timeout for TensorBoards and notebooks.
```

Comment:
non-blocking: Not to play favorites, but it feels like this is a highly requested feature and maybe we should put it higher up on the list? 😁

Reply:
Agreed with @justin-determined-ai 
In terms of user-facing new feature priority, I'd rank:
1. Support configuring an idle timeout for TensorBoards and Notebooks.
2. (All the workdir stuff in support of improved Notebook persistence and UX)
3. WebUI: Allow experiment owners to delete their own experiments.
4. WebUI: Add support for batch delete on the experiment list page.
5. WebUI: Display the latest log entry available for a trial at the bottom of the trial's page. ...
6. WebUI: Add support for displaying NaN and Infinity metric values.
7. Support the `MMDetection <https://github.com/open-mmlab/mmdetection>`__ library ...
8. (Everything else)

Reply:
Done!